src/LayerEditor/data/config.py

```diff
@@ -90,8 +90,7 @@ def update_current_workdir(self, file_name):
         directory = os.path.dirname(os.path.realpath(file_name))
         if self.workspace is not None and self.analysis is not None:
             analysis_dir = os.path.join(self.workspace, self.analysis)
-        if analysis_directory is None or directory != analysis_dir:
-            self.current_workdir = directory
+        self.current_workdir = directory
```

Comment:
There is a problem with analysis. It is not used at all, but I didnt know if it would be reintroduced in future, so I tried to preserve as much code as possible. Without this change it didnt work correctly.